travis-ymls/boto.travis.yml

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : boto
+# Source Repo         : https://github.com/boto/boto/
+# Travis Job Link     : https://travis-ci.com/github/asellappen/boto/
+# Created travis.yml  : No
+# Maintainer          : Arumugam N S <asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: python
+
+# Some versions of python only work on certain platforms. Specifying the distribution
+# to ensure the version of python will run without issue.
+matrix:
+  include:
+    # Trusty Tests
+    - os: linux
+      dist: trusty
+      python: "pypy"
+    - os: linux
+      dist: trusty
+      python: "2.6"
+    - os: linux
+      dist: trusty
+      python: "3.3"
+    - os: linux
+      dist: trusty
+      python: "3.4"
+    # Xenial Tests
+    - os: linux
+      dist: xenial
+      python: "2.7"
+    - os: linux
+      dist: xenial
+      python: "3.5"
+    - os: linux
+      dist: xenial
+      python: "3.6"
+    - os: linux
+      dist: xenial
+      python: "3.7"
+    - os: linux
+      dist: xenial
+      python: "pypy3.5"
+      env: 
+       - export CRYPTOGRAPHY_ALLOW_OPENSSL_102=1
```

Comment:
What is the reason for adding this env. Is it required for intel ?

Reply:
We can find the same error in  https://travis-ci.org/github/boto/boto also in base git .     We are linking against OpenSSL 1.0.2, which is no longer supported by the OpenSSL project. To use this version of cryptography we need to upgrade to a newer version of OpenSSL. For this version only we can also set the environment variable CRYPTOGRAPHY_ALLOW_OPENSSL_102 to allow OpenSSL 1.0.2,so that SSL is supported  for pypy3.5 in xenial dist.

Reply:
Pl. add the appropriate comment in the yml file

Reply:
if we add comments for each changes then entire yml will have more comments and earlier from community asked me to remove comments whatever i added for changes on yml file.

Reply:
@Siddhesh-Ghadi we are adding a new env variable here in the existing travis.yml. pl. confirm if it is okay to not add any comment for it. @asellappen for the failing test cases in intel you may want to add them to allowed_failures to make the build clean and also ensure there is an open issue for the same.

Reply:
@gururajrkatti , sounds good ,will be adding allowed_failures  for failing jobs under dist trust for amd64.

Reply:
Lets add comment for changes which are not common. `CRYPTOGRAPHY_ALLOW_OPENSSL_102=1` is not a common change so lets have it added in the file.

Reply:
Done. pl check